cmake/utils.cmake

```diff
@@ -123,18 +123,34 @@ function(combine_static_lib new_target target)
     message(FATAL_ERROR "MSVC does not yet support merging static libraries")
   endif()
 
-  set(all_libs "")
-  get_transitive_libs(${target} all_libs)
-  string(REPLACE ";" "\\\\naddlib " all_libs_string "${all_libs}")
+  # Should remove the if and use libtool on MacOS and Linux
```

Comment:
I kinda doubt libtool and cmake play well together.  Also, didn't libtool use to require only libraries that it has built itself?